### PR TITLE
Fixes #13556: another typo in rtf

### PR DESCRIPTION
**Pull request by Fdall**
https://www.rudder-project.org/redmine/issues/13556

### CODE REVIEW

File: rtf [hunk 1]
```diff
@@ -333,7 +333,7 @@ class Vagrant(Host):
     proxy = have_proxy()
     while "making sure that no vagrant up are running":
       (returncode, output) = shell("pgrep -f \"vagrant up\"", fail_exit=False)
-      if returncode != 0:
+      if returncode != 1:
```
Comment (peckpeck) [inline rtf]: you mean vagrant up should return 1 in case of succes ? that doesn't look unix

Comment (Fdall) [reply rtf]: it is pgrep that returns 1 when no process are found: https://www.gsp.com/cgi-bin/man.cgi?section=1&topic=pgrep

Comment (peckpeck) [reply rtf]: ok so the original ==0 was right

**Comment by Fdall** [comment]
Commit modified

**Pull request merged**